_includes/_sass/includes/_header.scss

```diff
@@ -98,85 +99,104 @@
 }
 
 .page-header-content {
-	// @extend %grid-context;
-	position: relative;
 	height: 6rem;
+	position: relative;
 }
 
 // handles layout and functionality for the header menus.
 // styling the internal bits happens in components/header/...
 .toggle-menu-icon {
 	@extend %menu-icon;
 	@include icon('menu');
+
+	background-color: $almost-black;
+	background-image: none;
 	display: block;
 	font-size: 1.3rem;
-	background-color: black;
 
 	@include breakpoint($breakpoint-bravo) {
 		display: none;
 	}
+
+	&:hover {
+		color: $color-accent-alternate-2;
+	}
 }
 
 .main-nav-supplemental-icon {
```

Comment:
The `:visited` state for this link is washing out the color, let's make sure that doesn't happen.

Reply:
Fixed